### PR TITLE
Adds Lots, Landownership & Map Saving Part 1

### PR DESCRIPTION
**Pull request by GeneriedJenelle**
Try two.

### CODE REVIEW

File: code/datums/lots/lots.dm [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+		// one more time, as some things that delete leave things behind.
+		for(var/obj/O in lot_area)
+			QDEL_NULL(O)
+//		SSmapping.maploader.load_map_tg(file(full_path), top_left.x, bottom_right.y, top_left.z, 1, 0)
+		SSmapping.maploader.load_map(file(full_path), top_left.x, bottom_right.y, top_left.z, 1, 0)
+
+		// Some things don't initialize at all after being loaded, it's weird, but this is needed too.
+		for(var/obj/O in lot_area)
+			O.persistence_save = FALSE
+			sleep(1)
```
Comment (ike709) [inline code/datums/lots/lots.dm]:
```suggestion
			CHECK_TICK
```

File: code/datums/lots/lots.dm [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+
+	var/lot_id		//associated lot ID
+	dont_save = FALSE
+
+/obj/effect/landmark/lotsign/initialize()
+	SSlots.lotsigns += src
+
+/obj/effect/landmark/lotsign/proc/get_lot_data()
+	if(!lot_id)
+		var/area/_area = loc.loc
```
Comment (ike709) [inline code/datums/lots/lots.dm]:
```suggestion
		var/area/_area = get_area(src)
```

File: code/modules/clothing/under/accessories/permits.dm [hunk 3]
```diff
@@ -92,3 +107,9 @@
 	icon = 'icons/obj/storage.dmi'
 	icon_state = "permit"
 	starts_with = list(	/obj/item/clothing/accessory/permit/gun = 8)
+
+/obj/item/weapon/storage/box/gun_permits/New()
+	for(var/instance in subtypesof(/obj/item/clothing/accessory/permit/gun))	// gives one of each permit, in a box.
+		starts_with += instance[1]
```
Comment (ike709) [inline code/modules/clothing/under/accessories/permits.dm]: This works?

Comment (GeneriedJenelle) [reply code/modules/clothing/under/accessories/permits.dm]: Doesn't. It's removed in the next commit that will be published tonight.

File: code/__defines/taxes_contraband.dm [hunk 4]
```diff
@@ -54,4 +66,4 @@
 		if(10000 to 79999)			return CLASS_MIDDLE
 		if(80,000 to INFINITY)		return CLASS_UPPER
```
Comment (ike709) [inline code/__defines/taxes_contraband.dm]:
```suggestion
		if(80000 to INFINITY)		return CLASS_UPPER
```

File: maps/~map_system/maps.dm [hunk 5]
```diff
@@ -110,6 +110,7 @@ var/list/all_maps = list()
 
 	var/council_email = "city-council@nanotrasen.gov.nt"
```
Comment (ike709) [inline maps/~map_system/maps.dm]: I assume these emails can't be changed, in which case all should be `var/static/`

Comment (GeneriedJenelle) [reply maps/~map_system/maps.dm]: In future map iterations I want to make it possible to have nanotrasen or non-nanotrasen maps, different colonies perhaps. These emails are created in memory on roundstart.

**Pull request merged**